packages/itinerary-body/src/defaults/line-column-content.tsx

```diff
@@ -65,14 +65,18 @@ export default function LineColumnContent({
     }
   );
 
+  const routeNumber = typeof route === "string" ? route : route?.shortName;
```

Comment:
I'm not a huge fan of calling it `routeNumber` since sometimes it will be a word or similar. How about `routeShortName`?

Reply:
Renamed in https://github.com/opentripplanner/otp-ui/pull/632/commits/a5c09f781f22711bdfef8409f61fa5d53d1703f9.